app/Http/Controllers/Api/ApplicationsController.php

```
@@ -3919,4 +3919,233 @@ private function validateDataApplications(Request $request, Server $server)
+                }
+                if ($request->has('host_path')) {
+                    $storage->host_path = $request->host_path;
+                }
+            } else {
+                if ($request->has('mount_path')) {
+                    $storage->mount_path = $request->mount_path;
+                }
+                if ($request->has('content')) {
+                    $storage->content = $request->content;
+                }
+            }
```

PHPMD (2.15.0)

[error] The method update_storage uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)